src/redux/migrations.ts

```diff
@@ -230,4 +230,11 @@ export default {
       wordByWordFontScale: initialState.quranReaderStyles.wordByWordFontScale,
     },
   }),
+  31: (state) => ({
+    ...state,
+    quranReaderStyles: {
+      ...state.quranReaderStyles,
+      quranFont: QuranFont.TajweedV4,
+    },
+  }),
```

Comment:
If this is applied, it will change every single user's Musahf to Tajweed! We need to check if the user's current setting is Tajweed first before applying this.

Reply:
Right! will add the condition